Makefile

```diff
@@ -57,6 +57,13 @@ dev: docker-test-base
 		-v $$GOPATH/bin:/home/fossa/go/bin \
 		quay.io/fossa/fossa-cli-test-base /bin/bash
 
+.PHONY: dev-osx
+dev-osx: docker-test-base
+	docker run --rm -it \
+		-p 2345:2345 \
```

Comment:
Is this for the debugger? Can you add this to all the Docker invocations in a separate PR?